docs/wallet/wallet-token.md

```diff
@@ -0,0 +1,74 @@
+---
+id: create-a-fungible-token
+title: Web Wallet Tokens
+---
+
+## **Introduction**
+
+**ESDT** stands for *Elrond Standard Digital Token*.
+
+Elrond network natively supports the issuance of custom tokens, without the need for contracts such as ERC20, but addressing the same use-cases.
+You can create and issue the ESDT token from [Elrond web wallet](https://wallet.elrond.com/) in a few steps. Let's go over these steps.
+
+## **Prerequisites**
+
+- A wallet on Elrond Network.
+- 0.05 EGLD issuance fee
+- fees for transactions
+
+## **Creating a fungible token from Web Wallet**
+
+To get started, open up the [Elrond web wallet](https://wallet.elrond.com/). You can create a new wallet if you do not have one or import your existing wallet. Here is a [guide](https://docs.elrond.com/wallet/web-wallet/) to help you navigate.
+
+Navigate to **ISSUE** section and click on **Tokens**.
+
+![issue-token}](/wallet/wallet-tokens/issue-token.png)
+
+:::note
+Web Wallet will handle the preparation of the transaction. Therefore, if you'd want a token with a supply of 10 and 2 decimals, you should simply put 10 as supply and 2 as number of decimals.
+:::
+
+When creating a token, you are required to provide the token name, a ticker, the initial supply, and the number of decimals for display purposes. 
```

Comment:
Can we change perhaps a little bit, so that it's more obvious that only the number of decimals are for display purposes, but not the other parameters? Also, from our point of view the decimals are only for display purposes, but perhaps for some integrators they could have a deeper significance.

Reply:
removed `for display purposes` as it is explained in the referenced URL below